### DISCUSSION TITLE
install as `--save-dev` or not?

### DISCUSSION THREAD

**Discussion by nimo23** [kubb-labs/kubb#1449] [Q&A] [answered]
Can I use 

`npm install @kubb/cli @kubb/core --save-dev`

instead of 

`npm install  @kubb/cli @kubb/core`

or is Kubb **not** allowed to be installed as a `dev` dependency?

If `save-dev` can be used, this should be stated in the docs (https://kubb.dev/getting-started/quick-start#install-kubb).

If `save-dev` can be also used when installing plugins (for example: https://kubb.dev/parsers/parser-ts#installation), then this should be stated in the docs.

**Comment by stijnvanhulle** [comment]
We added just the install command, in the end it's up to the user/team using Kubb to decide if they want to have Kubb as dependency or devDependency. The preferred way would be as devDependency but we don't want to make that strict or part of the documentation.

**Reply by nimo23** [reply]
> in the end it's up to the user/team using Kubb to decide if they want to have Kubb as dependency or devDependency

Many would just adopt it as is, without realizing that it is actually sufficient to install it as `dev` dependency. Normally, kubb should decide which dependencies are required to be installed as a **non** `dev` dependency.

**Reply by stijnvanhulle** [reply, marked as answer]
@nimo23 Let me add this to the documentation so it's clear that the prefered way should be dev.
